[DOCS] Prebuilt rules prerequisites: Add new section to What's New in 8.3 (#2327)
elastic/security-docs: docs/whats-new.asciidoc

15 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.3.0.

*Prerequisites and setup guides for Elastic prebuilt rules*
// The link below will need to be updated once PR elastic/security-docs#2069 is backported to 8.3 and merged (it will create the #rule-requisites section anchor). Until then, this will omit the section reference to allow CI to pass and create a preview.
// Full path should be: {security-guide}/rules-ui-management.html#rule-prerequisites[prerequisites]

> Elastic prebuilt rules now provide additional information to help you identify, install, and configure a rule's {security-guide}/rules-ui-management.html[prerequisites]. You can confirm these requirements in the *Related integrations* and *Required fields* fields on a rule's details page, and consult the rule's *Setup guide* for additional guidance.

Reviewer: ```suggestion
Elastic prebuilt rules now provide additional information to help you identify and meet their {security-guide}/rules-ui-management.html[prerequisites]. You can confirm these requirements in the *Related integrations* and *Required fields* sections of a rule's details page, and consult its *Setup guide* for additional guidance.
```
